Makefile

```diff
@@ -85,11 +85,16 @@ fallthrough: submodules
 manifests:
 	@$(WARN) Deprecated. Please run make generate instead.
 
-generate: $(HELM) $(KUSTOMIZE) go.vendor go.generate gen-kustomize-crds gen-install-doc
-	@$(OK) Finished vendoring and generating
+CRD_DIR = cluster/crds
 
+crds.clean:
+	@$(INFO) cleaning generated CRDs
+	@find $(CRD_DIR) -name '*.yaml' -exec sed -i.sed -e '1,2d' {} \; || $(FAIL)
+	@find $(CRD_DIR) -name '*.yaml.sed' -delete || $(FAIL)
+	@$(OK) cleaned generated CRDs
```

Comment:
Will these `sed` commands work on both OS X and Linux?

Reply:
@negz yep these were verified (after some trial and error :sweat_smile:) in the providers https://github.com/crossplane/provider-aws/blob/c7c185f2dc480396d86c1342af82af89151bba55/Makefile#L70

Hoping to also fix this in the parser -- but it is unfortunately a bit less straightforward than it seems.